Sources/CocoaLumberjack/DDFileLogger.m

```diff
@@ -1392,8 +1412,16 @@ - (void)lt_logData:(NSData *)data {
     }
 }
 
+- (id <DDFileLogMessageSerializer>)lt_logFileSerializer {
+    if ([_logFileManager respondsToSelector:@selector(logMessageSerializer)]) {
+        return _logFileManager.logMessageSerializer;
+    } else {
+        return [[DDFileLogPlainTextMessageSerializer alloc] init];
+    }
+}
+
 - (NSData *)lt_dataForMessage:(DDLogMessage *)logMessage {
-    NSAssert([self isOnInternalLoggerQueue], @"logMessage should only be executed on internal queue.");
```

Comment:
I've spotted this wrong assert message a couple of weeks ago, but deemed it not worth an extra PR just to correct it. Great you've spotted this, too :)